versions/0_3_rc/flake.nix

```diff
@@ -0,0 +1,28 @@
+{
+  inputs =
+    {
+      holochain = {
+        url = "github:holochain/holochain/holochain-0.3.0-beta-dev.48";
+        flake = false;
+      };
+
+      lair = {
+        url = "github:holochain/lair/lair_keystore-v0.4.4";
+        flake = false;
+      };
+
+      # holochain_cli_launch
+      launcher = {
+        url = "github:holochain/launcher/holochain-0.3";
+        flake = false;
+      };
+
+      # holochain_scaffolding_cli
+      scaffolding = {
+        url = "github:holochain/scaffolding/holochain-weekly";
```

Comment:
`holochain-0.3` branch is already availble on the scaffolding repo

https://github.com/holochain/scaffolding/tree/holochain-0.3